addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/internal/AllPlayBindingProperties.java

```diff
@@ -17,25 +17,34 @@
  */
 public class AllPlayBindingProperties {
 
-    private final Integer rewindSkipTimeInSec;
-    private final Integer fastForwardSkipTimeInSec;
+    private final double rewindSkipTimeInSec;
```

Comment:
This bug can be worked around when giving a pattern of `\d+` for the parameter like `<parameter name="myInt" type="integer" pattern="\d+">`. PaperUI will respect the pattern and validate the input.